### ISSUE TITLE
[QA] Unapplied security updates for build container

### ISSUE THREAD

**Issue by conorsch** [freedomofpress/securedrop#3574]
## Description

Same as #3551, essentially. The build container has unapplied security updates, causing builds to fail. We must update the build container in order to continue with 0.8.0 QA, specifically building 0.8.0~rc3 debs.

## Steps to Reproduce
1. Checkout `0.8.0-rc3` tag.
2. Run `make build-debs`

## Expected Behavior

Build succeeds without error.

## Actual Behavior

Build fails due to security upgrade check:

```
    =================================== FAILURES ===================================
    _________ test_ensure_no_updates_avail[docker://trusty-sd-sec-update] __________
    [gw3] linux2 -- Python 2.7.13 /home/conor/.virtualenvs/sd/bin/python2
    
    host = <testinfra.host.Host object at 0x34f19c7b310>
    
        def test_ensure_no_updates_avail(host):
            """
                Test to make sure that there are no security-updates in the
                base builder container.
            """
        
            # Filter out all the security repos to their own file
            # without this change all the package updates appeared as if they were
            # coming from normal ubuntu update channel (since they get posted to both)
            host.run('egrep "^deb.*security" /etc/apt/sources.list > /tmp/sec.list')
        
            dist_upgrade_simulate = host.run('apt-get -s dist-upgrade '
                                             '-oDir::Etc::Sourcelist=/tmp/sec.list '
                                             '|grep "^Inst" |grep -i security')
        
            # If the grep was successful that means security package updates found
            # otherwise we get a non-zero exit code so no updates needed.
    >       assert dist_upgrade_simulate.rc != 0
    E       AssertionError: assert 0 != 0
    E        +  where 0 = CommandResult(command='apt-get -s dist-upgrade -oDir::Etc::Sourcelist=/tmp/sec...ntu3.3] (1:5.14-2ubuntu3.4 Ubuntu:14.04/trusty-security [all])\n', stderr=None).rc
```

## Comments

We have lovely docs thanks to @msheiny: https://docs.securedrop.org/en/latest/development/dockerbuildmaint.html. Follow those, fix, submit PR.

**Comment by conorsch** [comment]
Due to the unresolved #2942, I'm going to push a fix up for this directly to `release/0.8`. After the 0.8.0 release, we can backport the change to develop, resolving conflicts manually if necessary

**Comment by redshiftzero** [comment]
So I just ran `make build-debs` on develop, and all tests were passing for me, so I suspect that this is exactly the same issue as #3551, and the trouble is just that the change in #3552 was never merged into `release/0.8` in #3554 due to the rebase-on-develop logic in CI in #2942

**Comment by redshiftzero** [comment]
closed by #3582
